custom/icds_reports/static/icds_reports/js/spec/prevalence-of-severe-report.directive.spec.js

```diff
@@ -14,25 +21,8 @@ describe('Prevalence Of Severe Directive feature flag disable', function () {
     pageData.registerUrl('icds_locations', 'icds_locations');
 
     beforeEach(module('icdsApp', function ($provide) {
-        $provide.constant("genders", [
-            {id: '', name: 'All'},
-            {id: 'M', name: 'Male'},
-            {id: 'F', name: 'Female'},
-        ]);
```

Comment:
This stuff is copy/pasted in a whole bunch of tests so i pulled it out into a few functions